apps/meteor/client/views/admin/rooms/EditRoom.tsx

```diff
@@ -97,7 +97,7 @@ const EditRoom = ({ room, onChange, onDelete }: EditRoomProps) => {
 	const handleArchive = useArchiveRoom(room);
 
 	const handleUpdateRoomData = useEffectEvent(async ({ isDefault, roomName, favorite, ...formData }) => {
-		const data = getDirtyFields(formData, dirtyFields);
+		const data = getDirtyFields({ isDefault, roomName, favorite, ...formData }, dirtyFields);
```

Comment:
Case for updating `roomName` is also handled below, having it here would override that. I would suggest instead of overriding the roomName, better delete roomName from `data` object.